mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -33,7 +33,7 @@ public void afterProjectsRead(MavenSession session) throws MavenExecutionExcepti
 
         configuration = Configuration.load();
 
-        if (configuration.isDisabled()) {
+        if (configuration.isDisabled() && configuration.isSkipTests()) {
```

Comment:
`isSkipTestsSet` reflects better what is the use case here. It doesn't have to follow the artificial javabean spec.